data/json/items/bionics.json

```diff
@@ -965,7 +965,7 @@
     "copy-from": "bionic_general_faulty",
     "type": "BIONIC_ITEM",
     "name": "Noisemaker",
-    "description": "A malfunctioning bionic.  When it was plugged in it would occasionally emit a loud burst of noise.",
+    "description": "A malfunctioning bionic.  When installed, it occasionally emits a loud burst of noise.",
```

Comment:
It's not supposed to be installed back, the idea here was that when it was still inside of you it was making noise.
Maybe replace `installed` with `powered`?